test/node-api/test_reference_by_node_api_version/test_reference_by_node_api_version.c

```diff
@@ -0,0 +1,175 @@
+#define NAPI_EXPERIMENTAL
```

Comment:
The `NAPI_VERSION` should have been defined in the binding.gyp.

```suggestion
```

Reply:
Good catch! It was a bug after I merged two tests. Fixed.